### PR TITLE
[WIP] Use common components for bind pages.

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
remove filters that were added to common (isBindingReady, statusCondition, and isServiceInstanceReady)

Depends on updated common version to include https://github.com/openshift/origin-web-common/pull/58

@benjaminapetersen

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
@jeff-phillips-18 heh, we gonna collide re: my PR #1505

File: app/scripts/directives/bindService.js [hunk 1]
```diff
@@ -71,28 +71,24 @@ function BindService($filter,
...
-      });
-    }
-    ctrl.steps.push({
-      label: 'Results',
-      id: 'results',
-      view: 'views/directives/bind-service/results.html'
-    });
+    ctrl.serviceSelection = {};
+    var formStepId = (ctrl.target.kind === 'Instance') ? 'applications' : 'services';
+    var formStepLabel = (ctrl.target.kind === 'Instance') ? 'Applications' : 'Services';
```
Comment (benjaminapetersen) [inline app/scripts/directives/bindService.js]: For the label, shall we just `_.upperFirst(formStepId)`?

File: app/scripts/filters/resources.js [hunk 2]
```diff
@@ -1090,25 +1090,6 @@ angular.module('openshiftConsole')
       return lastFinishTime;
     };
   })
-  // gets the status condition that matches provided type
```
Comment (benjaminapetersen) [inline app/scripts/filters/resources.js]: Makes sense to move these.

File: app/scripts/directives/bindService.js (hunk 1 above)
Comment (spadgett) [inline app/scripts/directives/bindService.js]: I think it makes sense as a separate property for when we need to translate

**Pull request closed without merging**